src/core.jl

```diff
@@ -223,7 +244,7 @@ function run(n::Int=typemax(Int))
                     rethrow()
                 else
                     bt = catch_backtrace()
-                    msgval.onerror(msgval.node, msgval.value, CapturedException(err, bt))
+                    msgval.onerror(msgval.node, msgval.value, node, CapturedException(err, bt))
```

Comment:
The docstring for `push!` needs to be updated to specify the new arguments for `onerror`. This is a breaking change (the next tag will have to be 0.4.0), as anyone using `onerror` now needs to change their callback functions. Unfortunately, I don't see how one can emit a deprecation warning, except possibly to look for this in failures of the `try`. It also needs to be advertised somehow, e.g., in a `NEWS.md` file.